### ISSUE TITLE
Inline images failing to load on outlook.live.com

### ISSUE THREAD

**Issue by ranmas** [trockerapp/trocker#18]
What is the outlook domain that you are using? Is it the normal consumer one (https://outlook.live.com/mail)?

Yes, I am using https://outlook.live.com/mail

Does this issue happen for all emails or only for some?

This happens for most emails, If I send a email with a inline image to myself it doesn't get blocked.  But for things like newegg newsletters, and even invoices from amazon  and other websites are blocked.

Are inline images blocked or is it only attachments? Could you include a screenshot?

This seems to effect only inline image.  Also there is no trocker button to let blocked images load.  The only thing that seems to work is totally disabling trocker.  This has been an issue for quite a few months now.  Images have been attached.

![trocker disabled](https://user-images.githubusercontent.com/7015459/137427300-48d65f28-ec97-424a-8871-8acb41fc742f.jpg)
![Trocker issue](https://user-images.githubusercontent.com/7015459/137427304-c1f05e88-6e5d-4bf6-80d6-814bff5cad52.jpg)
![trocker newegg](https://user-images.githubusercontent.com/7015459/137427307-3f273274-53ad-408e-9f4e-490bfd310ff6.jpg)
s.

**Comment by ranmas** [comment]
Here are a couple of inspector screenshots

In this image trocker is blocking a squre enix header logo. from  https://cdn-prod.scalefast.com/public/assets/client/123622/prj_503/mails/images/header.jp
![trocker blocked](https://user-images.githubusercontent.com/7015459/137429236-e51bbf8d-5043-4ede-80c7-a4428cd81760.jpg)
g

Then here Trocker is blocking the newegg sales deals from the inspector
![trocker inspect 1](https://user-images.githubusercontent.com/7015459/137429366-34c6a2eb-48d3-4931-bafc-4cd95798fd81.jpg)
.

**Comment by trockerapp** [comment]
Thank you for reporting the issue. We have been able to reproduce this issue, which seems to have been due to under the hood changed in the outlook UI. The issue should be [fixed](https://github.com/trockerapp/trocker/commit/2f5e0c677ff17eb65da7358ec3fbd2bf28599397) in version 2.6.16 that is rolling out on Firefox now and should be rolling out on Chrome webstore as well as soon as it is approved. Let us know if you still encounter issues after the update.
